### ISSUE TITLE
RME Web Remote Control

### ISSUE THREAD

**Issue by NOMNOMNOM117** [bitfocus/companion-module-requests#908]
Hoping someone with more knowledge using OSC and HTTP POST requests can make a module for the Web Remote from RME (Some instructions on how to use a terminal to control the device are demonstrated here https://docs.rme-audio.com/12mic/150-1c_remote_overview/). Specifically for the M-1610 Pro however I presume it would likely be easy to expand the module to support the rest of RME's range that use the web remote as they will mostly be the same. Trouble is its an expensive piece of kit so not many people have access to one for testing.

**Comment by josephdadams** [comment]
Is that the same as this request? #42

**Comment by NOMNOMNOM117** [comment]
@josephdadams No, the Web Remote is different from TotalMix. It is specifically for RME's Dante and Audio Video Bridging enabled product line (M-32 AD Pro, M-32 DA Pro, 12Mic, 12Mic-D, AVB Tool, and M-1610 Pro).
